modules/hdf5_handler/bes-testsuite/bes.cfdmr.conf.in

```diff
@@ -316,4 +316,6 @@ H5.DiskCacheFloatOnlyComp=true
 H5.DiskCacheCompThreshold=2.0
 H5.DiskCacheCompVarSize=10000
 
+H5.EscapeUTF8Attr = false
```

Comment:
Yes. Odd. It was set to true in my copy of the bes.cfdmr.conf file. I'm not sure how that could have worked with the CI/CD build. The conf files are not part of git according to 'git ls-files' I'll keep going though because maybe the issues in the next part will shed some light ...

Reply:
This is a local one. The bes.cfdmr.conf is actually generated by bes.cfdmr.conf.modules.in. In that file, the BES key is set to true. So your bes.cfdmr.conf is correct.